tests/integration/NewLexemeForm.test.ts

```diff
@@ -30,6 +32,7 @@ describe( 'NewLexemeForm', () => {
 				plugins: [ store ],
 				provide: {
 					[ ConfigKey as symbol ]: {},
+					[ MessagesKey as symbol ]: new Messages( new DevMessagesRepository() ),
 					[ ItemSearchKey as symbol ]: new DevItemSearcher(),
 					[ LanguageCodesProviderKey as symbol ]: {
 						getLanguageCodes: () => [ 'en', 'en-ca', 'de' ],
```

Comment:
Just randomly saw this while looking through the diff – this looks wrong now? (And I guess we should check other `getLanguageCodes` fake implementations too.)

Reply:
GitHub won’t let me comment on an unaffected line, but L94/97 looks similar. That’s the only one I found so far, though.